index.html

```diff
@@ -165,8 +166,62 @@ <h2 id="about-title">About me</h2>
        <li>Gitlab</li>
        <li>Terminal</li>
      </ul>
+   </div>
+   </section>
+
+     <div class="contact">
+<p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+<form action="https://formspree.io/f/mzbokdyd"  method="post" id="form">
+<fieldset>
+  <input type="text" id="name" placeholder="Full Name" maxlength="30" required>
+  <input type="email" id="email" placeholder="Email Address" required>
+  <textarea name="message" id="message" cols="30" rows="4" maxlength="500" required>Enter Text Here</textarea>
+</fieldset>
+
+  <button id="contact-button" class="resume-button project-button"  type="submit" name="button">Get in touch</button>
```

Comment:
- [ ] Kindly increase the padding and remove the border of your submit button to make it close to the Figma design.